akkadotnet/getakka.net
# Fix Github Edit Links

Resolves #121.

## Problem Description
`docs.hbs` was using `{{ basename }}` (an assemble.io variable) in a hard-coded path, but that doesn't allow for the concept of sub-folders, as `basename` only contains the file name without extension.

As we discovered in analysis of #121, we need to make use of the directory as well as the base name.

## Solution: Dirname and replacement using a custom Handlebars helper
I updated the URL to make use of `dirname` (another assemble.io variable), but realized it's using the generated dirname which uses the output folder of `/web`, which I needed to revise to read `/src`. 

It doesn't appear that there are any built-in text replacement helpers for handlebars, so I created one and added it via `helper-replace.js` in the assemble.io helper options.

## To Test
* Build locally
* Explore docs pages and make sure that the "edit on GitHub" link works
* Check out http://127.0.0.1:8080/docs/working-with-actors/creating-actors for a specific example of a sub-page that is working now.
